app/views/games/show.html.erb

```diff
@@ -40,3 +40,4 @@
   <% end %>
 </section>
 
+<%= button_to "Leave Game", action: "leave_game", slug: @game.slug %>
```

Comment:
Not sure if this should be part of this PR or if you want to move on to something else, but if we are doing this nice clear-up stuff after clicking 'leave game', it would be nice to ensure the same happens if they leave the game by another method, e.g. by clicking back in the browser. I'll put it on the todos - unless you fancy doing a little JS that alerts the player when they try clicking back?